metricbeat/module/system/service/services.go

```diff
@@ -0,0 +1,123 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
Nit. I guess these files wanted to be called `service.go` and `service_test.go` (as the metricset) :slightly_smiling_face: